[{"raw":{},"required":["Filename","Size"],"package":"python3-pytest-instafail","source":"pytest-instafail","version":"0.4.2-3","section":"universe/python","priority":"optional","architecture":"all","depends":["python3-pytest","python3:any"],"suggests":["python-pytest-instafail-doc"],"installedSize":33,"maintainer":"Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>","description":"plugin for py.test that shows failures and errors instantly","homepage":"https://github.com/pytest-dev/pytest-instafail","filename":"pool/universe/p/pytest-instafail/python3-pytest-instafail_0.4.2-3_all.deb","size":5998,"md5":"739d365718de7a8d341e01fed4b65b98","sha1":"cdf73d777fbe6f0ad59cfabf0f1ba0fa02580a6d","sha256":"788bfb38e2794f3c26877f64f2414e0f9e583073c2e3d47801f0666475cafcaa","sha512":"048c4bb4592f3da929ef284b4808da30d2aaa88103f1907441cd01968988a4e78bb7b48f72469ff566645f5f2356329093b89bcf7c33047f67c07b0166aa2c28","descriptionMd5":"4acabc61b3293224ea4a27802dc75041"}]